From dd05fdce72e5549b7407293580d12a4651b304a7 Mon Sep 17 00:00:00 2001
From: Jeff Kudrick <jeff.kudrick@freescale.com>
Date: Wed, 28 Nov 2012 10:34:46 -0600
Subject: [PATCH 106/602] ENGR00235193 - GPS Support for i.MX53

To enable the latest GPS sensor logic, a new directory is created
to house the date needed by the Orion binaries.  In testing,
addition of the GMS binaries required an increase in the system
image size.  Finally, although the gpsreset.sh file is not used
in the Orion configuration file, moved the binary to the correct
directory for Jelly Bean.

Signed-off-by: Jeff Kudrick <jeff.kudrick@freescale.com>
---
 imx5x/BoardConfigCommon.mk | 2 +-
 imx5x/imx53_smd.mk         | 2 +-
 imx5x/init.rc              | 1 +
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/imx5x/BoardConfigCommon.mk b/imx5x/BoardConfigCommon.mk
index efaeb83..45c5368 100755
--- a/imx5x/BoardConfigCommon.mk
+++ b/imx5x/BoardConfigCommon.mk
@@ -66,5 +66,5 @@ INTERNAL_USERIMAGES_USE_UBIFS=true
 TARGET_USERIMAGES_SPARSE_EXT_DISABLED := true
 TARGET_PROVIDES_INIT_RC := true
 
-BOARD_SYSTEMIMAGE_PARTITION_SIZE := 209715200
+BOARD_SYSTEMIMAGE_PARTITION_SIZE := 293601280
 BOARD_FLASH_BLOCK_SIZE := 4096
diff --git a/imx5x/imx53_smd.mk b/imx5x/imx53_smd.mk
index c039243..3855ae2 100644
--- a/imx5x/imx53_smd.mk
+++ b/imx5x/imx53_smd.mk
@@ -12,7 +12,7 @@ PRODUCT_COPY_FILES += \
 	device/fsl/imx53_smd/fstab.freescale:root/fstab.freescale \
 	device/fsl/imx53_smd/init.freescale.rc:root/init.freescale.rc \
         device/fsl/imx53_smd/vold.fstab:system/etc/vold.fstab \
-	device/fsl/imx53_smd/gpsreset.sh:system/etc/gpsreset.sh \
+	device/fsl/imx53_smd/gpsreset.sh:system/bin/gpsreset.sh \
 	device/fsl/common/input/eGalax_Touch_Screen.idc:system/usr/idc/eGalax_Touch_Screen.idc \
 	device/fsl/common/input/eGalax_Touch_Screen.idc:system/usr/idc/HannStar_P1003_Touchscreen.idc 
 
diff --git a/imx5x/init.rc b/imx5x/init.rc
index b470988..f1e4d83 100644
--- a/imx5x/init.rc
+++ b/imx5x/init.rc
@@ -205,6 +205,7 @@ on post-fs-data
     mkdir /data/app-private 0771 system system
     mkdir /data/app-asec 0700 root root
     mkdir /data/app 0771 system system
+    mkdir /data/app/ing 0771 root root
     mkdir /data/property 0700 root root
     mkdir /data/ssh 0750 root shell
     mkdir /data/ssh/empty 0700 root root
-- 
1.8.0

